.eslintignore

```diff
@@ -7,3 +7,4 @@ test-results/
 .svelte-kit/
 demo/vite-env.d.ts
 demo/src/app.d.ts
+**/environment.prod.ts
```

Comment:
Rather than excluding this file, I would prefer adding the file to the list of files in `tsconfig.app.json`.